boards.txt

```diff
@@ -15,222 +15,77 @@
 # Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 
 
-adafruit_feather_m0.name=Adafruit Feather M0
-adafruit_feather_m0.vid.0=0x239A
-adafruit_feather_m0.pid.0=0x800B
-adafruit_feather_m0.vid.1=0x239A
-adafruit_feather_m0.pid.1=0x000B
-adafruit_feather_m0.upload.tool=bossac
-adafruit_feather_m0.upload.protocol=sam-ba
-adafruit_feather_m0.upload.maximum_size=262144
-adafruit_feather_m0.upload.use_1200bps_touch=true
-adafruit_feather_m0.upload.wait_for_upload_port=true
-adafruit_feather_m0.upload.native_usb=true
-adafruit_feather_m0.build.mcu=cortex-m0plus
-adafruit_feather_m0.build.f_cpu=48000000L
-adafruit_feather_m0.build.usb_product="Feather M0"
-adafruit_feather_m0.build.usb_manufacturer="Adafruit"
-adafruit_feather_m0.build.board=SAMD_FEATHER_M0
-adafruit_feather_m0.build.core=arduino
-adafruit_feather_m0.build.extra_flags=-DARDUINO_SAMD_ZERO -D__SAMD21G18A__ {build.usb_flags}
-adafruit_feather_m0.build.ldscript=linker_scripts/gcc/flash_with_bootloader.ld
-adafruit_feather_m0.build.openocdscript=openocd_scripts/arduino_zero.cfg
-adafruit_feather_m0.build.variant=feather_m0
-adafruit_feather_m0.build.variant_system_lib=
-adafruit_feather_m0.build.vid=0x239A
-adafruit_feather_m0.build.pid=0x800B
-adafruit_feather_m0.bootloader.tool=openocd
-adafruit_feather_m0.bootloader.file=feather/samd21_sam_ba.bin
+mcci_catena_4410.name=Catena4410
+mcci_catena_4410.vid.0=0x040E
+mcci_catena_4410.pid.0=0x0018
+mcci_catena_4410.upload.tool=bossac
+mcci_catena_4410.upload.protocol=sam-ba
+mcci_catena_4410.upload.maximum_size=262144
+mcci_catena_4410.upload.use_1200bps_touch=true
+mcci_catena_4410.upload.wait_for_upload_port=true
+mcci_catena_4410.upload.native_usb=true
+mcci_catena_4410.build.mcu=cortex-m0plus
+mcci_catena_4410.build.f_cpu=48000000L
+mcci_catena_4410.build.usb_product="Catena 4410"
+mcci_catena_4410.build.usb_manufacturer="MCCI"
+mcci_catena_4410.build.board=SAMD_FEATHER_M0
+mcci_catena_4410.build.core=arduino
+mcci_catena_4410.build.extra_flags=-DARDUINO_SAMD_ZERO -D__SAMD21G18A__ {build.usb_flags}
+mcci_catena_4410.build.ldscript=linker_scripts/gcc/flash_with_bootloader.ld
+mcci_catena_4410.build.openocdscript=openocd_scripts/arduino_zero.cfg
+mcci_catena_4410.build.variant=arduino_zero
```

Comment:
This should be either Catena4410 or feather, not arduino_zero, I think. And yes, we should have a variant directory for each board.